staging/src/k8s.io/apiserver/pkg/util/flowcontrol/apf_controller.go

```diff
@@ -292,20 +292,9 @@ func newTestableController(config TestableConfig) *configController {
 }
 
 // MaintainObservations keeps the observers from
-// metrics.PriorityLevelConcurrencyObserverPairGenerator from falling
+// metrics.PriorityLevelConcurrencyObserverPairVec from falling
```

Comment:
We can get rid of this method completely from the interface now, right?
I'm assuming we will clean this up in a follow up right?

Reply:
Exactly.